yb-voyager/cmd/importDataState.go

```diff
@@ -55,17 +55,24 @@ metainfo/import_data_state/table::<table_name>/file::<base_name>:<path_hash>/
 	batch::<batch_num>.<offset_end>.<record_count>.<byte_count>.<state>
 */
 type ImportDataState struct {
-	exportDir string
-	stateDir  string
+	exportDir               string
+	stateDir                string
+	inProgressTaskImporters map[int]fileTaskImportStatusChecker
```

Comment:
add some comments here to explain this a bit and mention that this field is just for reading the status of tasks from `TaskImporter` for reference.